This is empiric side result of debugging Wacom WCOM5020 device.
Broken delayed reports better to wait between retries.

Signed-off-by: Denis Kaganovich <mahatma@eu.by>

--- a/drivers/hid/wacom_sys.c	2017-12-05 16:48:12.754154537 +0300
+++ d/drivers/hid/wacom_sys.c	2017-12-05 17:39:16.541000510 +0300
@@ -25,9 +25,11 @@
 static int wacom_get_report(struct hid_device *hdev, u8 type, u8 *buf,
 			    size_t size, unsigned int retries)
 {
-	int retval;
+	int retval = 0;
 
 	do {
+		if (retval)
+			msleep(100);
 		retval = hid_hw_raw_request(hdev, buf[0], buf, size, type,
 				HID_REQ_GET_REPORT);
 	} while ((retval == -ETIMEDOUT || retval == -EAGAIN) && --retries);
@@ -42,9 +44,11 @@ static int wacom_get_report(struct hid_d
 static int wacom_set_report(struct hid_device *hdev, u8 type, u8 *buf,
 			    size_t size, unsigned int retries)
 {
-	int retval;
+	int retval = 0;
 
 	do {
+		if (retval)
+			msleep(100);
 		retval = hid_hw_raw_request(hdev, buf[0], buf, size, type,
 				HID_REQ_SET_REPORT);
 	} while ((retval == -ETIMEDOUT || retval == -EAGAIN) && --retries);
